src/ui/public/styles/base.less

```diff
@@ -254,6 +254,22 @@ kbn-table, .kbn-table {
     }
   }
 
+  td {
+    position: relative;
+    .table-cell-filter {
+      position: absolute;
+      background-color: rgba(255, 255, 255, 0.75);
+      white-space:nowrap;
+      right: 0;
+      visibility: hidden;
+    }
+  }
+
+  td:hover {
+    .table-cell-filter {
+      visibility: visible;
+    }
+  }
```

Comment:
Then can you add these styles to this file, to tweak the appearance of the filter icons? The icons should end up looking like this:

![image](https://cloud.githubusercontent.com/assets/1238659/21702997/db2ad884-d363-11e6-8710-c49e04ec7a7f.png)

```less
/**
 * 1. Align icon with text in cell.
 */
.docTableRowFilterIcon {
  font-size: 14px;
  line-height: 1; /* 1 */
  
  & + & {
    margin-left: 5px;
  }
}
```